apps/frameworksTest/src/App.css

```
@@ -4,8 +4,8 @@
   text-align: center;
   background: linear-gradient(135deg, #667eea 0%, #764ba2 100%);
   color: white;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto',
-    sans-serif;
+  font-family:
+    -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', sans-serif;
```

Stylelint (17.14.1)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)